tap_jira/streams.py

```diff
@@ -128,6 +129,28 @@ def update_user_date(page):
         page['userStartDate'] = transform_user_date(page['userStartDate'])
 
     return page
+
+class BoardsAgile(Stream):
```

Comment:
Is this being used at all? I noticed in the the data-houston [PR](https://github.com/degreed/data-houston/pull/2089/files), we are not using this `BoardsAgile` stream

Reply:
Yeah, I left it in because after getting the BoardsAgile to work, I was able to get the BoardsGreenhopper to work. So I reckon it's nice for future usage.

Reply:
Yeah, I left it in because after getting the BoardsAgile to work, I was able to get the BoardsGreenhopper to work. So even though it's not necessary, I reckon it's ready for future usage.

Reply:
I've added the BoardsAgile.json to this branch, and updated the catalog.json in the branch of our own work repository